notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -106,29 +106,7 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.                ov_model = OVModelForCausalLM.from_pretrained(fp16_model_dir, compile=False,
</pre><p>there is no need in adding cache_dir as cache_dir used for compilation, you create model with compile=False, so caching will not performed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
removed